xml/obs_build_job_constraints.xml

```diff
@@ -3,276 +3,462 @@
+Constraint: hardware:cpu:flag power8
+%endif</screen>
+   <para>or</para>
+   <screen>%if "%_repository" == "images"
+Constraint: hardware:disk:size unit=M 4000
+%endif</screen>
+   <para> For a full treatment of constraint syntax, see
+    <xref linkend="_constraint_syntax"/>. </para>
+  </sect2>
+  <sect2>
+   <title>Package-scoped constraints</title>
+   <para> Setting constraints at the package level is achieved by including
+    an XML file called <literal>_constraints</literal> in the package sources.
+    The Build Service will attempt to validate this file when it is committed
+    (from the osc command line) or saved (in the web UI) to prevent invalid XML
+    from reaching the Build Service. </para>
+   <para> Here is a simple example showing what a <literal>_constraints</literal>
+    file might look like: </para>
+   <screen>&lt;?xml version="1.0"?&gt;
+&lt;constraints&gt;
+  &lt;hardware&gt;
+    &lt;physicalmemory&gt;
+      &lt;size unit="M"&gt;2000&lt;/size&gt;
+    &lt;/physicalmemory&gt;
+    &lt;disk&gt;
+      &lt;size unit="G"&gt;5&lt;/size&gt;
+    &lt;/disk&gt;
+  &lt;/hardware&gt;
+  &lt;sandbox&gt;kvm&lt;/sandbox&gt;
+  &lt;hostlabel exclude="true"&gt;SLOW_CPU&lt;/hostlabel&gt;
+&lt;/constraints&gt;</screen>
+   <para> For details on constraint qualifiers and how to specify them
+    in a <literal>_constraints</literal> file, see
+    <xref linkend="_constraint_syntax"/>. </para>
+  </sect2>
+  <sect2>
+   <title>Build recipe-scoped constraints</title>
+   <para> At the build recipe level, constraints are set by embedding lines
+    containing constraint qualifiers and values directly in RPM spec files or
+    Dockerfiles. Such lines take the form </para>
+    <screen>#!BuildConstraint: &lt;QUALIFIER&gt; &lt;VALUE&gt;</screen>
+   <para> The <literal>#</literal> character at the beginning of the line
+    causes the build recipe parser (RPM, Docker, podman) to ignore the
+    whole line, but in combination with the <literal>!</literal> character
+    signifies an OBS-specific directive that is picked up by a pre-processor
+    within the OBS back-end. </para>
+   <para> Instead of specifying subqualifiers by nesting
+    directives like in XML, colons are used. For example: </para>
+   <screen>#!BuildConstraint: linux:version:min 3.0</screen>
+   <para> In this example, "linux:version:min" is the constraint qualifier and
+    "3.0" is the value. </para>
+   <para> As described for project-scoped conditionals, above,
+    <literal>#!BuildConstraint</literal> lines can be guarded with various
+    conditionals to limit their effect to certain architectures
+    or, e.g., multibuild flavors. </para>
+  <note><para>
+   Be careful when setting build constraints. The idea is to use them to
+   express minimum values for the various parameters, below which builds
+   are likely to fail. If you set a constraint too high, you risk reducing the
+   pool of compliant build workers down to a very low number, or even to zero.
```

Comment:
Maybe add that this will lead to a failed job, if the OBS has no worker qualifing for it.